onecodex/distance.py

```diff
@@ -102,6 +102,19 @@ def unifrac(self, weighted=True, rank=Rank.Auto):
         tree = self.tree_build()
         tree = self.tree_prune_rank(tree, rank=ocx_rank)
 
+        # `scikit-bio` requires that the tree root has no more than 2
+        # children, otherwise it considers it "unrooted".
+        #
+        # https://github.com/biocore/scikit-bio/blob/f3ae1dcfe8ea88e52e19f6693d79e529d05bda04/skbio/diversity/_util.py#L89
+        #
+        # Our taxonomy root regularly has more than 2 children, so we
+        # add a fake parent of `root` to the tree here.
+        from skbio.tree import TreeNode
+
+        new_tree = TreeNode(name="fake root")
+        new_tree.rank = "no rank"
+        new_tree.append(tree)
```

Comment:
Should `new_tree` be passed into the `beta_diversity` call below? And can we add a test for this?

Reply:
Resolved in 963de54